src/types.ts

```diff
@@ -66,7 +66,6 @@ export interface IndexedNode extends Node {
 export type OrderedFlow = Array<IndexedNode>;
 
 export interface NormalizedNode extends IndexedNode {
-  component: string;
```

Comment:
I have removed this since a lookup of the enum directly is better in code (i.e. `ComponentType.Question` instead of "Question")

For the stored data structure, serializing this as a string would be the nicest way to handle this in the future since the value would then also be human readable. We may also want to transition `ComponentType` to use an object "as const" instead of a numeric enum.